packages/lesswrong/lib/collections/posts/schema.js

```diff
@@ -88,11 +88,11 @@ const schema = {
     optional: true,
     viewableBy: ['guests'],
     onInsert: (post) => {
-      return Utils.slugify(post.title);
+      return Utils.getUnusedSlugByCollectionName("Posts", Utils.slugify(post.title))
```

Comment:
Duplicate post slugs are allowed; they're a purely decorative part of the URL, only the ID counts. So this change isn't needed and in rare cases (duplicate or highly similar post names) makes the URLs uglier.

Reply:
Not true, since the collection-pages are grabbing posts by slug

Reply:
This has already caused some errors in the past

Reply:
Also, this PR makes `getUnusedSlugByCollectionName` require an `oldSlugs` field, which this doesn't have.